modules/contentbox/modules/contentbox-admin/handlers/authors.cfc

```diff
@@ -332,7 +338,7 @@ component extends="baseHandler"{
         flash.put( "authorID", rc.authorID );
         prc.oAuthor = authorService.get( id=rc.authorID );
 
-        if ( prc.oCurrentAuthor.getAuthorID() != prc.oAuthor.getAuthorID() ) {
+        if ( prc.oCurrentAuthor.isLoaded() && prc.oCurrentAuthor.getAuthorID() != prc.oAuthor.getAuthorID() ) {
```

Comment:
why did you add the `isLoaded()`?

Reply:
isLoaded check tells us if a user is authenticated but not logged in. We need that flag to know if the user is who is trying to log in or if this user is an admin who is already logged in. We can discuss the strategy if this is not very clean.

Reply:
But `getAuthorID()` will be empty, so no need for a second test. Did you test with an invalid user id?